core/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -219,13 +222,17 @@ private TachyonWorker(InetSocketAddress masterAddress, InetSocketAddress workerA
       WorkerService.Processor<WorkerServiceHandler> processor =
           new WorkerService.Processor<WorkerServiceHandler>(mWorkerServiceHandler);
 
-      mServerTNonblockingServerSocket = new TNonblockingServerSocket(workerAddress);
-      mPort = NetworkUtils.getPort(mServerTNonblockingServerSocket);
-      mServer =
-          new TThreadedSelectorServer(new TThreadedSelectorServer.Args(
-              mServerTNonblockingServerSocket).processor(processor)
-              .selectorThreads(selectorThreads).acceptQueueSizePerThread(acceptQueueSizePerThreads)
-              .workerThreads(workerThreads));
+      mServerTServerSocket = new TServerSocket(workerAddress);
+      mPort = NetworkUtils.getPort(mServerTServerSocket);
+
+      mServer = new TThreadPoolServer( new TThreadPoolServer.Args(
```

Comment:
Extra space, and I think it might be neater to fit it into one line if possible?